notebooks/265-qrcode-monster/265-qrcode-monster.ipynb

```diff
@@ -0,0 +1,971 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import qrcode
</pre><p>please add some description before this function or/and move it closer to its usage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
moved the code block